### ISSUE TITLE
repo not tagged

### ISSUE THREAD

**Issue by KrisThielemans** [ismrmrd/ismrmrd#222]
Last tag is 1.8.0, but version is much later than that.

We rely on tags to grab versions, so would it be possible to tag a recent version (and automatically tag, I guess)?

**Comment by hansenms** [comment]
Yeah, we have some inconsistencies. Some repos (e.g. https://github.com/ismrmrd/ismrmrd-python) we use a release workflow to trigger production of packages (conda packages) and others we just make the package and push it in the CI pipeline and there is no tag and no release, which makes it less obvious what the latest release is. I will bring this up on our ISMRMRD/MRD group meeting on Monday and come up with a consistent strategy. And yes, it should be automated, relying on manual tags is not the best.

**Comment by hansenms** [comment]
We are doing this now.
